### DISCUSSION TITLE
Search Node behind NAT

### DISCUSSION THREAD

**Discussion by horwedja** [Security-Onion-Solutions/securityonion#7168] [Unsupported Versions]
Starting to feel a bit silly. Here is my setup.
ManagerNode: Pubilc IP. all looks great
Searchnode: Behind Nat with a local IP. Setup goes great. accepted minion via firewall. appears in in grid as OK. IP is display as the local IP though.
Downhill from here.
Kibana: No matching indices found: No indices match pattern "*:so-*"
Is seems that the Search => Manager works ports (9300, 9696)
It also seems that Manger => Search is looking for local IP. I may be wrong because i cant run so-allow or so-firewall on search nodes.
Found post about base URL but nope.
Does search nodes run behind a NAT?

**Comment by dougburks** [comment]
The manager needs to be able to initiate connections directly to the search node. You may want to consider VPN.
